subseas_flex_fcst/config-sample.yaml

```diff
@@ -12,10 +12,15 @@ loglevel: info
 # Forecast options
 y_transform: true
 forecast_mu_file: CFSv2_SubXPRCP_CCAFCST_mu_Apr_Apr-1-2022_wk1.txt
+forecast_mu_filePattern: CFSv2_SubXPRCP_CCAFCST_mu_*.txt
 forecast_var_file: CFSv2_SubXPRCP_CCAFCST_var_Apr_Apr-1-2022_wk1.txt
+forecast_var_filePattern: CFSv2_SubXPRCP_CCAFCST_var_*.txt
 obs_file: CFSv2_SubXPRCP_CCAFCST_obs_Apr_Apr-1-2022_wk1.txt
+obs_filePattern: CFSv2_SubXPRCP_CCAFCST_obs_*.txt
 hcst_file: CFSv2_SubXPRCP_CCAFCST_xvPr_Apr_Apr-1-2022_wk1.txt
+hcst_filePattern: CFSv2_SubXPRCP_CCAFCST_xvPr_*.txt
 variable: Precipitation Anomaly
+tp_length: 6
```

Comment:
This is currently not used. I feel it will be at some point, but then it should be 7 rather than 6. More intuitive for a forecast predicting a week.

Reply:
If we keep it, I would prefer to call it target_period_length.

Reply:
I do currently use this in the code when calculating target start / end. But if there is a better way to do this calculation then I will change it.

For now I kept it in the config file but changed the name to be more descriptive.